### DISCUSSION TITLE
[RFC] View naming convention in SwiftUI

### DISCUSSION THREAD

**Discussion by markgravity** [nimblehq/ios-templates#477] [Polls]
## Why
It is inconsistent in the way we name a view when using SwiftUI, we can see the way how Apple names its views: 
- `VStack` not `VStackView`
- `List` not `ListView`
so it means we don't need to add the `View` postfix every time in our custom `View` or we need a commit in the team about this convention

**Comment by blyscuit** [comment]
I think it makes sense in Clean-MVVM to always have the `View` postfix so we can easily distinguish a SwiftUI view from a Domain Model but if there's a better way to differentiate Model and View we can drop the View postfix. 🤔

**Comment by phongvhd93** [comment]
I closed this RFC since it reached the due date. We will follow the 2nd option.

Thread on Slack: https://nimble-co.slack.com/archives/C418DBL2W/p1684289178823219
